domain/account.go

```diff
@@ -9,16 +9,21 @@ import (
 
 // account is the aggregate root for a bank account.
 type account struct {
+	// Opened is the opened state of the account.
```

Comment:
```suggestion
	// Opened is true if the account has been opened.
```

How about this? The other new bool parameters could be changed similarly.

Reply:
Yeah that sounds better than the generic docs I had. I'll have another look at the other aggregate existence bools when I change this.